demos/mcp_server_memory/db/sqlite_manager.py

```
@@ -0,0 +1,581 @@
+
+    def delete_memory(self, memory_id: str) -> bool:
+        """Delete a memory item from the database.
+
+        Args:
+            memory_id: The ID of the memory to delete
+
+        Returns:
+            bool: True if successful, False otherwise
+        """
+        try:
+            now = timestamp()
```

Ruff (0.11.9)

Local variable `now` is assigned to but never used

Remove assignment to unused variable `now`

(F841)